tests/ReplayData/Branch.testEditRequiredStatusChecksChecks.txt

```diff
@@ -0,0 +1,21 @@
+https
+PATCH
+api.github.com
+None
+/repos/jacquev6/PyGithub/branches/integrations/protection/required_status_checks
+{'Authorization': 'Basic login_and_password_removed', 'Content-Type': 'application/json', 'User-Agent': 'PyGithub/Python'}
+{"checks": [{"context": "check1"}, {"context": "check2", "app_id": -1}, {"context": "check3", "app_id": 123456}]}
+200
+[('status', '200 OK'), ('x-ratelimit-remaining', '4994'), ('content-length', '0'), ('server', 'nginx/1.0.13'), ('connection', 'keep-alive'), ('x-ratelimit-limit', '5000'), ('etag', '"81ba94a48ad867b26d48c023ac584f43"'), ('date', 'Mon, 07 May 2018 13:42:41 GMT'), ('content-type', 'application/json; charset=utf-8')]
+''
```

Comment:
looks like this is not correct

Reply:
how to create the file automatically? This does not work for me in macOS
`Branch.testEditRequiredStatusChecksChecks --record --auth_with_token`

```
(pygithub-dev) <user>@<computer> PyGithub % pytest -k Branch.testEditRequiredStatusChecksChecks --record --auth_with_token
=============================================================================================== test session starts ===============================================================================================
platform darwin -- Python 3.10.13, pytest-7.4.4, pluggy-1.3.0
rootdir: /Users/<user>/VSCode/python/PyGithub
configfile: pyproject.toml
plugins: github-actions-annotate-failures-0.2.0, subtests-0.11.0, cov-4.1.0
collected 890 items / 890 deselected / 0 selected                                                                                                                                                                 

============================================================================================= 890 deselected in 0.32s =============================================================================================
```